src/medCore/gui/toolboxes/medToolBox.cpp

```diff
@@ -64,6 +65,8 @@ medToolBox::medToolBox(QWidget *parent) : QWidget(parent), d(new medToolBoxPriva
     connect(d->body, SIGNAL(maximized()), this, SLOT(behaveWithBodyVisibility()));
 
     this->setSizePolicy(QSizePolicy::Minimum, QSizePolicy::Minimum);
+
+    connect(this, SIGNAL(displayError(int)), this, SLOT(handleDisplayError(int)));
```

Comment:
where is the connection between the `medJobItem::displayError` signal and this one?